packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts

```diff
@@ -77,12 +83,31 @@ export const getMenuItemStyles = ({ theme }: WfoTheme) => {
 
     const selectedSubMenuItem = css({
         ...baseSubMenuStyles,
+        height: `${theme.base * 2.25}px`,
+
+        backgroundColor: toSecondaryColor(theme.colors.mediumShade),
+        borderRadius: theme.border.radius.medium,
         fontWeight: theme.font.weight.medium,
         color: theme.colors.darkestShade,
+        marginLeft: `${theme.base / 4}px`,
+        paddingLeft: `${theme.base / 2}px`,
```

Comment:
Think size.xs (currently 4px) would be a more safe way

Reply:
Same